requirements.txt.freeze

```diff
@@ -0,0 +1,54 @@
+asn1crypto==1.3.0
```

Comment:
This is exactly what you see when running `pip freeze` in the virtualenv in prod, except for that the `mysqlclient` package entry is not here, but it's in the `-mysql.txt.freeze` file.